web/layout_attributes/nesting_flex_layouts/my_element.html

```diff
@@ -1,26 +1,26 @@
 <link rel="import" href="packages/polymer/polymer.html">
```

Comment:
I typically don't put any html imports in my elements any more, they aren't necessary (polymer.dart includes an @HtmlImport to this file). By using @HtmlImport exclusively you can ensure there are not canonicalization issues.